packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -200,7 +200,7 @@ export const BookEventFormChild = ({
       const { uid, paymentUid } = responseData;
       const fullName = getFullName(bookingForm.getValues("responses.name"));
       if (paymentUid) {
-        return router.push(
+        router.push(
```

Comment:
Be very clear that router.push returns a void value. It's possible this wasn't historically and this has changed with more recent Next.js versions.